web/cobrands/sass/_base.scss

```diff
@@ -1451,6 +1451,10 @@ html.js #map .noscript {
   top: 44px;
 }
 
+#hide_pins_link {
```

Comment:
I think being able to hide pins on the mobile map (even as and when pagination is added) is quite important!